### PR TITLE
Moving hardcoded messages to US and KR properties

### PR DESCRIPTION
**Pull request by chd830**
## Description
The hard-coded code in the jsp file under views.admin.common was added to en.properties and ko.properties.

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/main/resources/messages/message_ko_KR.properties [hunk 1]
```diff
@@ -12,6 +12,7 @@ msg.common.valid=입력칸을 모두 채우고 다시 시도해주세요.
 msg.common.valid2=에러가 발생했습니다. 다시 시도하거나 시스템 관리자에게 연결하세요.
 error.common.system=에러가 발생했습니다. 다시 시도하거나 시스템 관리자에게 연결하세요. <br /> {0}
 msg.common.success=성공적으로 완료되었습니다.
+msg.common.report.select.sheet=한 개 이상의 시트를 선택해 주세요.
```
Comment (chd830) [inline src/main/resources/messages/message_ko_KR.properties]:
```suggestion
```

File: src/main/resources/messages/message_en_US.properties [hunk 2]
```diff
@@ -53,13 +55,11 @@ msg.project.android.noticefile.required=Please upload Notice.html File.
...
 msg.distribute.confirm.reset=<b>This description is already registered on the Open Source Distribution Site.</b><br /><br />This distribution will only be removed in the FOSSLight System.<br />If you want to delete it from OSDD site, Click the DELETE WITH OSDD button<br /><br />Are you sure to reject the distribution?
 msg.distribute.confirm.reset.warn=This description can not be found on the Open Source Distribution Site.<br />Are you sure to reject the distribution?
 msg.distribute.model.required=Model information is required.
 msg.distribute.sync.addedmodel=There are models added to the OSDD system. <br/>Please check the model information and try again.<br/>
```
Comment (chd830) [inline src/main/resources/messages/message_en_US.properties]:
```suggestion
msg.distribute.sync.addedmodel=There are models added to the OSDD system. <br/>Please check the model information and try again.<br/>
msg.distribute.description.duplicate=The same category, same description already distributed in
msg.distribute.origin.link=<a class=tabLink href=#none onclick="createTabInFrame(''{0}_Distribute'', ''#/project/distribution/{0}'')">&lt;project id: {0}&gt;</a>
```

File: src/main/resources/messages/message_en_US.properties [hunk 3]
```diff
@@ -53,7 +55,7 @@ msg.project.android.noticefile.required=Please upload Notice.html File.
...
-msg.distribute.confrim.overwritefile=File(<b>{0}<b/>) with the same name already exists in the category. <span style="color:red;">Do you want to overwrite <b>{0}<b/> file?</span><br>(If you change the file, it applies to <span style="color:blue;">all relevant models that already distributed.</span>)
+msg.distribute.confrim.overwritefile = \ub3d9\uc77c\ud55c Category\uc5d0 \ub3d9\uc77c\ud55c \uc774\ub984\uc758 \ud30c\uc77c(<b>{0}<b/>)\uc774 \uc774\ubbf8 \uc874\uc7ac\ud569\ub2c8\ub2e4. <span style="color:red;"><b>{0}<b/> \ud30c\uc77c\uc744 \ub36e\uc5b4 \uc50c\uc6b0\uaca0\uc2b5\ub2c8\uae4c?</span><br>(\ub9cc\uc57d \ud574\ub2f9 \ud30c\uc77c\uc744 \ub36e\uc5b4\uc50c\uc6b0\uac8c \ub420 \uacbd\uc6b0, \ubcc0\uacbd \uc0ac\ud56d\uc740 <span style="color:blue;">\uc774\ubbf8 \ubc30\ud3ec\ub418\uc5c8\ub358 \ubaa8\ub378</span>\uc5d0\ub3c4 \uc801\uc6a9\ub429\ub2c8\ub2e4.)<br>File(<b>{0}<b/>) with the same name already exists in the category. <span style="color:red;">Do you want to overwrite <b>{0}<b/> file?</span><br>(If you change the file, it applies to <span style="color:blue;">all relevant models that already distributed.</span>)
```
Comment (epicarts) [inline src/main/resources/messages/message_en_US.properties]: 유니코드 없이 아래와 같이 나와야합니다. 수정 부탁드려요.

```suggestion
msg.distribute.confrim.overwritefile=File(<b>{0}<b/>) with the same name already exists in the category. <span style="color:red;">Do you want to overwrite <b>{0}<b/> file?</span><br>(If you change the file, it applies to <span style="color:blue;">all relevant models that already distributed.</span>)
```

File: src/main/resources/messages/message_ko_KR.properties [hunk 4]
```diff
@@ -20,6 +20,8 @@ msg.common.confirm.mergeAndSave=Identification이 변경되었습니다. 저장
 msg.common.search.check.date=기간을 다시 확인해주세요.
 #msg.common.notice_required=Notice 파일이 필요합니다.
 #msg.common.report.select.sheet=한 개 이상의 시트를 선택해 주세요.
+msg.common.cannot.delete=삭제 할 수 없습니다.
+msg.common.cannot.registered.delete=등록된 라이센스는 삭제가 불가능 합니다.
```
Comment (epicarts) [inline src/main/resources/messages/message_ko_KR.properties]: 라이센스 -> 라이선스로 수정 부탁드려요.

File: src/main/resources/messages/message_ko_KR.properties [hunk 5]
```diff
@@ -73,6 +75,8 @@ msg.oss.confirm.ossVersion=해당 오픈소스의 라이선스는 버전에 따
 msg.oss.nickname.exists=같은 OSS에 등록된 닉네임이 추가되었습니다. 닉네임을 확인하고 다시 시도해주세요.
 msg.oss.deactivated=비활성화된 오픈소스
 msg.oss.duplicated=중복된 오픈소스
+msg.oss.include.license=본 OSS는 여러 License 하의 Source Code를 포함하고 있습니다.
```
Comment (epicarts) [inline src/main/resources/messages/message_ko_KR.properties]: License => 라이선스, Source Code => 소스 코드

File: src/main/resources/messages/message_ko_KR.properties [hunk 6]
```diff
@@ -73,6 +75,8 @@ msg.oss.confirm.ossVersion=해당 오픈소스의 라이선스는 버전에 따
...
+msg.oss.select.license=본 OSS는 등록된 License 중 하나를 선택할 수 있습니다.
```
Comment (epicarts) [inline src/main/resources/messages/message_ko_KR.properties]: License => 라이선스 위와 같습니다~

**Review by epicarts — CHANGES_REQUESTED** [review]

**Review by epicarts — APPROVED** [review]

File: src/main/resources/messages/message_ko_KR.properties [hunk 7]
```diff
@@ -98,34 +111,5 @@ spdx.tag.failure=SPDX Tag 파일 생성에 실패하였습니다.
...
-msg.oss.select.synchronize=오픈소스 속성 동기화를 선택합니다.
-msg.oss.select.project=프로젝트를 선택하십시오.
-
-# statistics
-msg.statistics.required.date=날짜값을 입력하세요.
-msg.statistics.warn.daterange=날짜 범위 오류
-msg.statistics.notice.admin=관리자에게 문의하세요.
\ No newline at end of file
+enter.watcher.error=Watcher AD ID를 입력하세요.
+invalid.email.error=잘못된 이메일 주소입니다.
```
Comment (chd830) [inline src/main/resources/messages/message_ko_KR.properties]:
```suggestion
invalid.email.error=유효하지 않은 이메일 주소입니다.
```

File: src/main/resources/messages/message_ko_KR.properties [hunk 8]
```diff
@@ -98,34 +111,5 @@ spdx.tag.failure=SPDX Tag 파일 생성에 실패하였습니다.
...
-msg.oss.required.select=등록할 오픈소스를 선택하십시오.
-msg.oss.select.synchronize=오픈소스 속성 동기화를 선택합니다.
-msg.oss.select.project=프로젝트를 선택하십시오.
-
-# statistics
-msg.statistics.required.date=날짜값을 입력하세요.
-msg.statistics.warn.daterange=날짜 범위 오류
-msg.statistics.notice.admin=관리자에게 문의하세요.
\ No newline at end of file
+enter.watcher.error=Watcher AD ID를 입력하세요.
```
Comment (chd830) [inline src/main/resources/messages/message_ko_KR.properties]:
```suggestion
enter.watcher.error=Watcher의 AD ID를 입력하세요.
```

File: src/main/resources/messages/message_ko_KR.properties [hunk 9]
```diff
@@ -73,6 +76,16 @@ msg.oss.confirm.ossVersion=해당 오픈소스의 라이선스는 버전에 따
 msg.oss.nickname.exists=같은 OSS에 등록된 닉네임이 추가되었습니다. 닉네임을 확인하고 다시 시도해주세요.
 msg.oss.deactivated=비활성화된 오픈소스
 msg.oss.duplicated=중복된 오픈소스
+msg.oss.include.license=본 OSS는 여러 라이선스 하의 소스 코드를 포함하고 있습니다.
+msg.oss.select.license=본 OSS는 등록된 라이선스 중 하나를 선택할 수 있습니다.
+
+# Partner
+msg.partner.notice=이진 분석 결과는 참조 데이터로만 사용됩니다. 정확도가 향상될 때까지 BOM 및 OSS 공지사항에 포함되지 않습니다.
+msg.partner.notice.term=업로드된 펌웨어는 한 달 동안만 보관됩니다.
+msg.partner.warning=3rd party software 상태가 다른 사용자에 의해 변경되고 있습니다. 자세한 내용은 reviewer에게 문의하세요.
```
Comment (chd830) [inline src/main/resources/messages/message_ko_KR.properties]:
```suggestion
msg.partner.warning=Reviewer에게 문의하세요.
```

File: src/main/resources/messages/message_ko_KR.properties [hunk 10]
```diff
@@ -73,6 +76,16 @@ msg.oss.confirm.ossVersion=해당 오픈소스의 라이선스는 버전에 따
...
+msg.partner.confirm=댓글을 삭제하시겠습니까?
+msg.partner.delete.warning=파티를 삭제하시겠습니까?\n(데이터가 영구적으로 삭제됩니다.)
```
Comment (chd830) [inline src/main/resources/messages/message_ko_KR.properties]:
```suggestion
msg.partner.delete.warning=해당 데이터가 영구적으로 삭제됩니다. 그래도 진행하시겠습니까?
```

**Review by epicarts — APPROVED** [review]

**Pull request merged**